Mihai-Blaga/COMP30022
# AA-48 Add event without intent

## Changes

- Edited addEvent function from CalendarUtil
- AddNewMeetingFragment now calls addEvent() instead
- Returns the user to the previous page after adding Event

## How to test
- Try adding a new meeting, there should no longer be a popup to google calendar. Check if the event is added correctly added by exiting the app and checking calednar.
